docs/tutorials/azure-quantum/getting_started_honeywell.ipynb

```diff
@@ -0,0 +1,328 @@
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### List all targets\n",
+    "\n",
+    "You can now list all the targets that you have access to, including the current queue time and availability."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "source": [
+    "service.targets()"
+   ],
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "[<Target name=\"ionq.qpu\", avg. queue time=250 s, Available>,\n",
+       " <Target name=\"ionq.simulator\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1\", avg. queue time=0 s, Unavailable>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2\", avg. queue time=93509 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-sim\", avg. queue time=26 s, Available>]"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 3
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Run a simple circuit\n",
+    "\n",
+    "Now, let's create a simple Cirq circuit to run."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "source": [
+    "import cirq\n",
+    "\n",
+    "q0, q1 = cirq.LineQubit.range(2)\n",
+    "circuit = cirq.Circuit(\n",
+    "    cirq.H(q0), # Hadamard\n",
+    "    cirq.CNOT(q0, q1), # CNOT\n",
+    "    cirq.measure(q0, q1, key='b') # Measure both qubits into classical register \"b\"\n",
+    ")\n",
+    "circuit"
+   ],
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/html": [
+       "<pre style=\"overflow: auto; white-space: pre;\">0: ───H───@───M(&#x27;b&#x27;)───\n",
+       "          │   │\n",
+       "1: ───────X───M────────</pre>"
+      ],
+      "text/plain": [
+       "0: ───H───@───M('b')───\n",
+       "          │   │\n",
+       "1: ───────X───M────────"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 4
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "You can now run the program via the Azure Quantum service and get the result. The following cell will submit a job that runs the circuit with 100 shots, wait until the job is completed and return the results."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "source": [
+    "result = service.run(program=circuit, repetitions=100)"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "......."
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This returns a `cirq.Result` object. Note that we used the API validator, which only returns zeros."
```

Comment:
Nit: if possible we prefer to use 2nd person. Using things like "you can now do this thing" vs "we did this thing".